gradle/lib-admin.gradle

```diff
@@ -25,7 +66,12 @@ ext {
     hasLibAdminUi = this.&hasLibAdminUi
 }
 
-task flush( type: Delete, dependsOn: clean ) {
-    description = 'Clean the project from built sources and dependencies'
-    delete '.xp'
+afterEvaluate {
+    if ( hasTask( 'clean' ) )
+    {
+        task flush( type: Delete, dependsOn: clean ) {
```

Comment:
flush is rudimental. clean should be sufficient. this is likely the reason we still observe the problem